src/lib.rs

```diff
@@ -123,6 +123,9 @@ mod init;
 pub mod pbkdf2;
 
 pub mod rand;
+mod limb;
+pub use limb::GFp_rand_mod;
```

Comment:
I don't understand why this `pub use` is here. I think it should be removed. We actually don't want to expose this in the API.